module/tools/SystemConfiguration/src/SystemConfiguration.cpp

```diff
@@ -0,0 +1,363 @@
+#include "SystemConfiguration.h"
```

Comment:
.hpp and licence please

Reply:
For the licence, this code is coming from 4Tel, so i'm not 100% sure how we should handle it in these cases.
The best result I think would be the entire NUbots codebase going to MIT